pkgs/top-level/all-packages.nix

```
@@ -12054,6 +12054,8 @@ with pkgs;
 
   pywal = with python3Packages; toPythonApplication pywal;
 
+  pywb = callPackage ../tools/misc/pywb {};
```

```suggestion
  pywb = python3.pkgs.callPackage ../tools/misc/pywb {};
```

Please also avoid using `with python3.pkgs;` inside the derivation, and use the function arguments at the top of the file. This makes the derivation more overridable

---

bump